controllers/comment.controller.js

```diff
@@ -0,0 +1,23 @@
+const { selectCommentsByArticleId } = require('../models/comment.model');
+const { selectArticleById } = require('../models/article.model');
+
+
+exports.getCommentsByArticleId = (req, res, next) => {
+	const { article_id } = req.params;
+
+	Promise.all([
+		selectArticleById(article_id),
+		selectCommentsByArticleId(article_id)
+	])
+
+		.then(([article, comments]) => {
+			res.status(200).send({ comments });
+		})
+		.catch((err) => {
```

Comment:
This catch block isn't necessary here as the `article not found` custom error comes from the Promise.reject in the selectArticleById() controller will already be handled by your error handlers.